pkg/kubelet/container/helpers.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/tools/record"
+	resourcehelper "k8s.io/component-helpers/resource"
```

Comment:
For new imports, only use an alias if there is a conflict.

Same for other files.